drivers/pds/lib/utils.go

```diff
@@ -914,6 +914,30 @@ func CreatepostgresqlWorkload(dnsEndpoint string, pdsPassword string, scalefacto
 	return deployment, err
 }
 
+func CollectPodLogsandValidateWorkloads() error {
+	pods, err := k8sCore.GetPods("automation", nil)
+	if err != nil {
+		log.Errorf("An Error occured while getting pods")
+	}
+	failure_messages := []string{"Name or service not known", "could not translate host name", "Connection refused"}
+	for _, pod := range pods.Items {
+		if pod.Name == "pgload-slcww-64b4b7ffdd-bjt7w" {
```

Comment:
Are you sure this `pgload-slcww-64b4b7ffdd-bjt7w` always going to be a name for this pod?